conn.go

```diff
@@ -418,6 +418,7 @@ func (c *conn) mux() {
 				nextSession = newSessionResp{err: errorErrorf("reached connection channel max (%d)", c.channelMax)}
 				continue
 			}
+			debug(1, "NS(%d) new session", next)
```

Comment:
this log helped me track the servicebus issue that reached the maximum number of channels